frontend/public/components/utils/headings.tsx

```diff
@@ -119,6 +131,7 @@ export type PageHeadingProps = {
   title?: string | JSX.Element;
   titleFunc?: (obj: K8sResourceKind) => string | JSX.Element;
   customData?: any;
+  releaseBadgeType?: string;
```

Comment:
If you made this `badge: React.ReactNode` then the parent could simply supply a custom badge component of either `DevPreviewBadge` or `TechPreviewBadge` instead of mapping a string to a badge type in case we later have additional badges or badges that are linked to documentation or some other customization.

Reply:
@nemesis09 Better just remove the `ReleaseBadge` component and use individual `DevPreviewBadge` and `TechPreviewComponent` components as suggested by @christianvogt .